### ISSUE TITLE
Special characters and max characters per line?

### ISSUE THREAD

**Issue by SpartanPartyPoop** [Zombiefleischer/pause-text#29]
I noticed that a line of text won't be displayed if there is an `;` in the text. I'm also having problems with other lines of text, but I haven't figured out what the underlying problem is.

Are there other special characters that don't work or is there a maximum length of the text?

**Comment by Zombiefleischer** [comment]
I don't think that there is a maximum length of text. At some point, JavaScript crashes or gets an overflow, I think. I don't have a limit, since I figure that people use common sense and figure out, when a text length gets problematic.

> I noticed that a line of text won't be displayed if there is an `;` in the text.
Oh, I think I need to do some input sanitization 😅 I will work on a solution tomorrow.

**Comment by Zombiefleischer** [comment]
Can you provide me with more details to the `;` problem? Foundry Version, System, etc. I did a quick test, and it doesn't seem to cause issues in the two systems (black-flag and dnd5e) that I used.

**Comment by Zombiefleischer** [comment]
Closed due to inactivity.
